cmd/api/main.go

```diff
@@ -100,7 +98,9 @@ func main() {
 	var err error
 
 	dbURI := fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=%s sslmode=%s sslrootcert=%s", serverConfig.DB_HOST, serverConfig.DB_USER, serverConfig.DB_PASSWORD, serverConfig.DB_NAME, serverConfig.DB_PORT, serverConfig.DB_SSLMODE, serverConfig.DB_SSLROOTCERT)
-	gormConfig := &gorm.Config{}
+	gormConfig := &gorm.Config{NamingStrategy: schema.NamingStrategy{
```

Comment:
It is not readily clear from the code that this is how you set the Postgresql "schema" using gorm. Can you please add a comment with a link to the relevant docs in gorm?